config/jobs/kubernetes/sig-node/node-docker.yaml

```diff
@@ -28,6 +28,43 @@ periodics:
   annotations:
     testgrid-dashboards: sig-node-docker
     testgrid-tab-name: docker-node-conformance
+- name: ci-docker-node-conformance-kubetest2
+  interval: 2h
+  decorate: true
+  path_alias: k8s.io/kubernetes
+  extra_refs:
+  - org: kubernetes
+    repo: test-infra
+    base_ref: master
+    path_alias: k8s.io/test-infra
+  decoration_config:
+    timeout: 65m
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+  annotations:
+    testgrid-dashboards: sig-node-docker
+    testgrid-tab-name: docker-node-conformance-kubetest2
+  spec:
+    containers:
+    - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20211108-d0d3c6d7a8-experimental
```

Comment:
requests and limits?

Reply:
what type of requests and limits are you talking about? 🤔

Reply:
I'm not sure, I remember in one of previous PRs I saw the note that requests and limits will be needed on all containers (https://github.com/kubernetes/test-infra/pull/24039#issuecomment-962882732). but I don't know details

Reply:
Okay, so I didn't add them as there were no requests and limits present in the corresponding non-kubetest2 job.
I also don't know if it's needed.

cc: @ameukam @amwat

Reply:
The requests and limits are only required for jobs running on `k8s-infra-prow-build*` (aka field `cluster: k8s-infra-prow-build`) clusters. They can be added now or when sig-node prowjobs will be migrated to the community-owned infrastructure.